homeassistant/components/tts/__init__.py

```diff
@@ -55,6 +56,43 @@
 CONF_SERVICE_NAME = "service_name"
 CONF_TIME_MEMORY = "time_memory"
 
+CONF_DESCRIPTION = "description"
+CONF_FIELDS = "fields"
+CONF_TARGET = "target"
+
+SAY_FIELDS = {
```

Comment:
We should maybe load this from `service.yaml`

Reply:
I agree. 

This all feels pretty hacky. Would it make more sense to just tell the frontend to show the `say` service description and call it a day?

Reply:
Right now the frontend doesn't have access to the say service description, as that service is not registered. For notity it is (notify.notify)

Reply:
Ah, we only include descriptions for registered services. That makes sense (and also meh).

Reply:
Changed it a tiny bit, to load it from the service file.